packages/admin-ui-docs/docs/guidelines/components/alert.mdx

```diff
@@ -175,6 +175,58 @@ function Example() {
 
   </Panel>
   <Panel id="designReference">
+
+## Documentation
+
+Alerts are notifications of mild to high priority. They may inform the user about events they should know, or explain a problem and point out a solution. They may be triggered by a user action or not, and may refer to actions performed by other users, accounts, on different moments in time.
```

Comment:
This paragraph can be removed for all components, since it's content is already in the overview tab.